### DISCUSSION TITLE
개발자님 혹시 예약 신청 버튼 활성화 되면 신청 하는 부분이 코드로 어느 부분인가요?

### DISCUSSION THREAD

**Discussion by ranware** [changdoc/naver-vaccine-macro#119] [Q&A]
개발자님 혹시 예약 신청 버튼 활성화 되면 신청 하는 부분이 코드로 어느 부분인가요?

조금 전에 잔여 떻는데 신청버튼 활성화 되고 버튼이 안눌려져서 실패로 떻는데..

**Comment by changdoc** [comment]
https://github.com/changdoc/naver-vaccine-macro/blob/main/content.js 174라인 보시면됩니다.

근데 실패로 떴다는건 예약 신청이 진행된거 아닌가요?

**Reply by ranware** [reply]
아 제가 알람을 듣고 화면을 보니 멈춰져 있었습니다. 그래서 제가 눌렸더니... 실패가 됬었어요...

**Reply by changdoc** [reply]
그렇군요. 혹시 테스트 모드에 체크가 되어 있는건 아니죠?
![image](https://user-images.githubusercontent.com/1879494/127167788-46bcc5d8-8191-48c9-b850-f27ad817f2b9.png)

**Reply by ranware** [reply]
> 그렇군요. 혹시 테스트 모드에 체크가 되어 있는건 아니죠?
> ![image](https://user-images.githubusercontent.com/1879494/127167788-46bcc5d8-8191-48c9-b850-f27ad817f2b9.png)

기본으로 체크 되어 있지 않나용? 그대로 썻었어요... 아니면 제 컴퓨터만 잠깐 렉인거 같기도 하네요..

**Reply by changdoc** [reply]
기본값은 체크 해제 상태이긴 합니다만
체크가 되어있었다면 예약 신청은 진행 하지 않습니다

**Comment by ranware** [comment]


**Reply by changdoc** [reply]
내일 저도 한번 확인해볼 수 있도록 하겠습니다.
